beat/models/distributions.py

```diff
@@ -137,6 +137,16 @@ def multivariate_normal_chol(
 
     return logpts
 
+##Mahdi
+def cumulative_normal(x, s=num.sqrt(2)):
+    # Cumulative distribution function for the standard normal distribution
+    return 0.5 + 0.5 * tt.erf(x/s)
+
+def polarity_llk(observed_polarities, pred_amplitudes, gamma, sigma):
```

Comment:
again naming, you compare observed_polarities with pred_amplitudes, 
so decide for one naming, either both polarities or both amplitudes ...
sorry for being so pedantic but in few monthes we will not know anymore whats going on in the code, 
these programming naming conventions are really important for code maintenance!

Reply:
I was making mistakes, sorry, 
the true formula is now in the code.

Reply:
still naming ...see above